src/Test/Utilities/Portable/Compilation/TestOperationWalker.cs

```diff
@@ -421,6 +421,24 @@ public override void VisitAnonymousObjectCreationExpression(IAnonymousObjectCrea
             base.VisitAnonymousObjectCreationExpression(operation);
         }
 
+        public override void VisitObjectOrCollectionInitializerExpression(IObjectOrCollectionInitializerExpression operation)
+        {
+            base.VisitObjectOrCollectionInitializerExpression(operation);
+        }
+
+        public override void VisitMemberInitializerExpression(IMemberInitializerExpression operation)
+        {
+            base.VisitMemberInitializerExpression(operation);
+        }
+
+        public override void VisitCollectionElementInitializerExpression(ICollectionElementInitializerExpression operation)
+        {
+            var addMethod = operation.AddMethod;
```

Comment:
The behavior of this override and a few above doesn't add anything to the base behavior. They can be removed.

---
In reply to: [126534989](https://github.com/dotnet/roslyn/pull/20689#discussion_r126534989) [](ancestors = 126534989)